tests/issues.targets

```diff
@@ -52,6 +52,13 @@
         </ExcludeList>
     </ItemGroup>
 
+    <!-- All Windows targets -->
+    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and '$(TargetsWindows)' == 'true'">
```

Comment:
This should be all targets instead of just windows. However, we should still continue to run this test on Jenkins. Can we pass through an environment variable for running under Helix?

Reply:
Talked with @jashook offline. It seems easier just to fix the test than introducing one more variable for "build against Helix". Reverting superpmicollect related changes and merging as it is.